Source/WebCore/style/StyleBuilderConverter.h

```diff
@@ -483,6 +483,8 @@ inline LengthPoint BuilderConverter::convertPositionOrAuto(BuilderState& builder
 {
```

Comment:
Should this function be called `convertPositionOrAutoOrNormal` ?

Reply:
Fixed.